estate_account/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# A custom real estate module, created for learning purposes
+
+{
+    'name': '[TUTO] Estate_Account',
+    'summary': 'Module to invoice sold real estate properties',
+    'depends': [
+        'base_setup', 'estate', 'account'
```

Comment:
'base_setup' here is not needed, as it is already a parent of 'account' which this module depends on ;)